packages/react/src/components/Popup/Popup.tsx

```diff
@@ -112,6 +112,9 @@ export interface PopupProps
   /** Element to be rendered in-place where the popup is defined. */
   trigger?: JSX.Element
 
+  /** Trigger is responsible for being focusable */
+  triggerHandlesFocus?: boolean
```

Comment:
Jush sharing my thoughts. This `prop` seems to be only needed to define if set `tabindex` to trigger or not. Frankly, for me a bit confusing from the prop name what it is for. 
If this prop is `false`, the trigger will be set attribute `tabindex=0`, what will make it tabbable, nit just focusable.
As one of the options:
1. we can have a prop `shouldTriggerBeTabbable` and make it `true` by default
2. Or, utilize attribute `data-is-focusable`. If it set to "false" to trigger, in `popupBehavior` check it and do not set  tabindex. Though it needs proper documentation, but kind of consistency of using it (we use it for FocusZone)

Reply:
I like the first option, seems cleaner

Reply:
In my opinion the second option is better, because we already have a mechanism for this, why introduce a new one. If we continue with this new pattern, we will start adding tabbable props on different components, and it's not something we want to do...

Reply:
But that means that `data-is-focusable` prop would add `data-is-focusable` attribute as well as add tabindex. It would make that prop both handled and unhandled, do we really want that?